common.js

```diff
@@ -2938,6 +2938,69 @@ const allSoloCars = {
       'all' : ['ss', 'ssm', 'ssp'],
     }
   },
+  'Kia': {
+    'Stinger (V6 Turbo)': {
+      '2018': ['fs'],
+      '2019': ['fs'],
+      '2020': ['fs'],
+    },
+    'Stinger (4-cyl Turbo)': {
+      '2018': ['fs'],
+      '2019': ['fs'],
+      '2020': ['fs'],
+    },
+    'Forte & Forte Koup (2.0L & 2.4L) ': {
+      'all': ['hs']
+    },
+    'Forte5': {
+      '2014': ['hs'],
+      '2015': ['hs'],
+      '2016': ['hs'],
+      '2017': ['hs'],
+      '2018': ['hs'],
+    },
+    'Optima': {
+      'all': ['hs']
+    },
+    'Rio': {
+      '2012': ['hs', 'sts'],
+      '2013': ['hs', 'sts'],
+      '2014': ['hs', 'sts'],
+      '2015': ['hs', 'sts'],
+      '2016': ['hs', 'sts'],
+      '2017': ['hs', 'sts'],
+      '2018': ['hs'],
+      '2019': ['hs'],
+      '2020': ['hs'],
+    },
+    'Sephia': {
+      'all': ['hs']
+    },
+    'Spectra5': {
+      'all': ['hs']
+    },
+    'Forte & Forte Koup (non-turbo)': {
+      'all': ['hs']
+    },
+    'Forte & Forte Koup (Turbo)': {
+      'all': ['sts']
+    },
+    'Spectra (1.8L 4-cyl)': {
+      'all': ['fsp']
+    },
+    'Forte Koup': {
+      '2008': ['fsp'],
+      '2009': ['fsp'],
+      '2010': ['dsp', 'fsp'],
```

Comment:
You cannot be eligible for both `fsp` and `dsp` at the same time, since they are subclasses of the same parent class. If you want to, I can make a separate branch for you to submit your PR to and then I can do the work of merging these models together. I think ideally we'd have the Forte as one model, and the Forte Koup as another model, with turbo and non-turbo variants if needed.

Reply:
Yes, that will be the best way. 👍